tests/run_all_tests.py

```diff
@@ -148,6 +148,8 @@ def _execute_run_time_compare(self, actual_run_time):
                     csv_writer.writerow(['test_name', 'expected_run_time (seconds)', 'actual_run_time (seconds)'])
                 csv_writer.writerow([self.test_name, expected_run_time, f'{actual_run_time:.2f}'])
             csv_file.close()
+            if actual_run_time > expected_run_time * 5:
```

Comment:
make sure that expected run-time is not 0, so that we do not fail on some test that has not yet been added to expected runtime.